script.rb

```diff
@@ -0,0 +1,42 @@
+require 'csv'
+require "google_drive"
+require 'mechanize'
+require 'rails'
+require 'byebug'
+
+
+id_array = ['ID']
+
+id_array.each do |id|
+
+  pass, sp_url = 'PASS', "https://docs.google.com/spreadsheets/d/1mNAJZp0B0nLZ6dYuoMZuYDmIRUd_w8PiiY_NtEUFTW8/edit#gid=0" if id == 'ID'
+
+  start_date = (Time.now.beginning_of_month).to_i.to_s + "000"
```

Comment:
I want to take data from 8/1 to now.
But in fact, this script takes data from 7/31 to now.